sys/include/ztimer/config.h

```diff
@@ -104,17 +104,37 @@ extern "C" {
 #endif  /* MODULE_ZTIMER_PERIPH_RTT */
 
 /**
- * @brief   The minimum pm mode required for ZTIMER_USEC to run.
+ * @brief   The minimum pm mode required for ZTIMER_TIMER to run.
  */
-#ifndef CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE
-#define CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
+#ifndef CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE
+    #ifdef CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE
+        #define CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE
+    #else
+        #define CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
+    #endif
```

Comment:
```suggestion
#ifndef CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE
#  ifdef CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE
#     define CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE CONFIG_ZTIMER_USEC_REQUIRED_PM_MODE
#  else
#    define CONFIG_ZTIMER_TIMER_REQUIRED_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
#  endif
```

We don't really have style of preprocessor stuff covered by the coding convention, but it is IMO better to keep the style at least consistent with the other macros in the same source file.

The reasoning for keeping the `#` as first character is that this is actually required by the C standard (at least in older versions). Spaces between the `#` and the preprocessor statement are explicitly valid and ignored. I hove no idea why two spaces are used for indent instead of the usual four, but let's just stick with that choice for consistency.

Reply:
I will change these when the first Murdock report is ready